modules/sb-start-asb.adoc

```diff
@@ -25,5 +25,13 @@ If you do not yet see these Service classes, you can check the status of the fol
 
```

Comment:
I think that the content starting at line 22 either needs to be nested under the final step or become its own step. (And s/once/after)

Reply:
Thanks for the great suggestion @kalexand-rh. I added a "verify that the service broker" has started to put all of this under (and also for the similar template service broker module).

I believe I addressed all of your comments.

Reply:
Thanks @bergerhoffer. It looks good.